src/components/sporsmal/typer/periode-komp.tsx

```diff
@@ -20,15 +21,27 @@ type AllProps = SpmProps & PeriodeProps;
 
 const PeriodeKomp = ({ sporsmal, index, slettPeriode }: AllProps) => {
     const { setValue, errors, getValues } = useFormContext()
+    const [ datoer, settDatoer ] = useState<Date[]>([])
     const id = sporsmal.id + '_' + index
     const htmlfor = sporsmal.id + '_t_' + index
     const feilmelding = hentFeilmelding(sporsmal)
+    Norwegian.rangeSeparator = ' - '
 
     useEffect(() => {
         setValue(id, hentPeriode(sporsmal, index))
         // eslint-disable-next-line
     }, [ sporsmal ]);
 
+    const onValueUpdate = (selectedDates: Date[]) => {
```

Comment:
Har nødt til å holde styr på intern state til Flatpickr, fordi `useEffect`en over ikke oppdateres i tide for å få med oss begge datoene i perioden.